internal/pkg/docker/dockercompose/decompose.go

```diff
@@ -122,3 +134,32 @@ func unsupportedServiceKeys(service map[string]any, svcName string) (IgnoredKeys
 
 	return ignored, nil
 }
+
+// isolateEnvFiles manually handles the env_file key so that compose-go does not try to read the env files from the disk.
+func isolateEnvFiles(service map[string]interface{}) ([]string, error) {
+	envFileStr := struct {
+		EnvFile string `mapstructure:"env_file"`
+	}{}
+	err := mapstructure.Decode(service, &envFileStr)
```

Comment:
whats the difference between this and just doing something like:
```
envFile, ok := service["env_file"].(string)
```
i'd prefer not to add the dep if this is all it's doing...

Reply:
The situation is a little bit more complex for arrays but the required code is quite a bit less complex than I thought. Thanks for suggesting that I give it another look!